department.rb

```diff
@@ -0,0 +1,39 @@
+class Department
+
+  attr_reader :name
+
+  def initialize( name )
+    @name = name
+    @employees = []
+  end
+
+  def addEmployee( employee )
+    @employees << employee
+  end
+
+  def parseReviews
```

Comment:
As mentioned in class, you're using the JavaScript naming convention to name methods (e.g. parseReviews).  In Ruby, the convention is parse_reviews.